### PR TITLE
Guillermo Guevara

### PR DESCRIPTION
**Pull request by memoguevara**
I add my biography. I had some issues when trying to commit but I solved it by switching to the virtual R Studio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+
+
+### My biography
+
+My name is **Guillermo Guevara** and I am second year MPP at Harris. Before coming to Harris, I held several positions at the Mexican Federal Government.I am interested in:
```
Comment (bensoltoff) [inline README.md:5]: Nice use of bold Markdown!

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+- Policing
+- Crime prevention
+- Political Accountability
+
+[Here you can check a photo of my idol Andy Murray](https://en.wikipedia.org/wiki/Andy_Murray#/media/File:Andy_Murray_(44087043305).jpg)
```
Comment (deblnia) [inline README.md:11]: This is a great use of a URL to render an image! Make sure you can also render images from local paths -- it's a great way to learn about directories and file systems at least.

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |         X  |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Reproducibility**         |       X    |              |            |

Remarks:

* Great work! You seem to have the homework submission workflow down, and you've used Markdown syntax really well. 😄

File: README.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+
+
+### My biography
+
+My name is **Guillermo Guevara** and I am second year MPP at Harris. Before coming to Harris, I held several positions at the Mexican Federal Government.I am interested in:
+
+- Policing
+- Crime prevention
+- Political Accountability
+
```
Comment (jchon11) [inline README.md:10]: Good use of bullet points. You could also use sub-bullets if you wanted to add some more detail to your topics. Not that you need it, just a suggestion for if you wanted to play with Markdown.

File: README.md [hunk 4]
```diff
@@ -1 +1,13 @@
...
+[Here you can check a photo of my idol Andy Murray](https://en.wikipedia.org/wiki/Andy_Murray#/media/File:Andy_Murray_(44087043305).jpg)
+
```
Comment (jchon11) [inline README.md:12]: I didn't think of embedding an image link in the homework, so this was an interesting choice. On an aesthetic level, I think it fits the concise and clear style of your biography. On the other hand, I do think that embedding an image itself would have been better for the assignment.

File: README.md [hunk 5]
```diff
@@ -1 +1,13 @@
...
+[Here you can check my linkdl page](https://www.linkedin.com/feed/?trk=guest_homepage-basic_nav-header-signin)
```
Comment (jchon11) [inline README.md:13]: I like the use of hyperlink instead of just a link itself.

**Review by jchon11 — COMMENTED** [review]
Overall, I think you did a fantastic job. You managed to capture all the main requirements of the homework in a relatively short biography, which I think is great. The only thing I would have preferred is embedding an image in the biography itself, rather than just a link to an image.

File: README.md [hunk 6]
```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+
+
+### My biography
+
+My name is **Guillermo Guevara** and I am second year MPP at Harris. Before coming to Harris, I held several positions at the Mexican Federal Government.I am interested in:
+
+- Policing
```
Comment (bamartinez) [inline README.md:7]: Great job with the bullet points. Although it is different than the way I wrote the code out, it came out well!

File: README.md (hunk 1 above)
Comment (bamartinez) [inline README.md:5]: You created a header well and used a bolded your name well!

File: README.md (hunk 5 above)
Comment (bamartinez) [inline README.md:13]: I used the hyperlink and it took me to the homepage. However, you properly embedded the link onto the script, so great work!

**Review by bamartinez — COMMENTED** [review]
Overall, the code you utilized in the bio was great! The header was properly coded and you bolded your name effectively. You also coded lists differently from how I did it, so I am happy to learn something new! The links all worked and I enjoyed the picture of Andy Murray. In regards to the code, I have a couple of suggestions. Instead of embedding links for pictures, you can try using the image syntax on Mastering Markdowns on GitHub to create a path to your image! A picture of yourself would also have been a great addition. These are all suggestions and do not take away from your amazing work. Great job!

**Pull request closed without merging**